go.sum

```diff
@@ -4,3 +4,10 @@ github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc h1:cAKDfWh5Vpd
 github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc/go.mod h1:LOuyumcjzFXgccqObfd/Ljyb9UuFJ6TxHnclSeseNhc=
 github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf h1:qet1QNfXsQxTZqLG4oE62mJzwPIB8+Tee4RNCL9ulrY=
 github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf/go.mod h1:ybxpYRFXyAe+OPACYpWeL0wqObRcbAqCMya13uyzqw0=
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
```

Comment:
If you run `go mod tidy` (and `go mod vendor`) it will mark indirect packages by adding a comment. I guess you didn't add all of these directly, right?

Reply:
I guess you are talking about `go.mod`, the `indirect` comment is in place there.

Reply:
Ok now I got it, `go-spew` should be there marked as indirect.
But I already removed `go.mod`,`go.sum` and `vendor` and ran the commands again. The result is the same, no change to what is committed.

Reply:
Okay, should be fine then. Exact behaviour is described in this brand new blog post: https://blog.golang.org/using-go-modules